src/scripts.js

```diff
@@ -1 +1,107 @@
-console.log('Hello world');
+//const recipeData = require("../data/recipes");
+// Get the modal
+
+//const recipeData = require("../data/recipes");
+
+//const ingredientsData = require("../data/ingredients");
+
+//const ingredientsData = require("../data/ingredients");
+let modal;
+function openModals(event) {
+  let eventId = event.target.id;
+  modal = document.getElementById(eventId + 'modal');
+  modal.style.display = 'block';
+}
+
+// Get the button that opens the modal
+var btn = document.getElementById("myBtn");
+
+// Get the <span> element that closes the modal
+var span = document.getElementsByClassName("close");
+
+
+// query selectors
+const recipeSection = document.querySelector('.body__main__section');
+
+// render recipes
+const recipeInstantiation = createRecipes();
+const ingredientHashmap = createIngredientHash();
+
+window.onload = displayHandler;
+
+// //When the user clicks anywhere outside of the modal, close it
+// window.onclick = function(event) {
+//   if (event.target == modal) {
+//     modal.style.display = "none";
+//   }
+// }
+
+
+
+
+function displayHandler() {
+  showRecipes();
+  const allModals = document.getElementsByClassName("body__main__section__article__button");
+  for (let i = 0; i < allModals.length; i++) {
```

Comment:
Very nice, dynamic, way to add and hide elements. 
Potential reasons to revisit
* Is this a case where for loops are valid
  * Can use forEach if not
* We can break this out into two separate add and hide functions
* Would it be better to assign our querySelector to a variable